checkpoint/cp-resource-management/README.md

```diff
@@ -47,10 +58,10 @@ In order to get the [Check Point ClougGuard](https://secure.dome9.com/) Cluster
 
 Refer to [values.yaml](values.yaml) for the full run-down on defaults. These are a mixture of Kubernetes and CloudGuard directives that map to environment variables.
 
-Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
+Specify each parameter using `--set key=value[,key=value]` or `--set-string key=value[,key=value]` to `helm install`. For example,

```

Comment:
since it's not helm documentation, I would leave only the first syntax.

Reply:
Do you want to mention it's relevant not only to `helm install` but to `helm upgrade `too?